tests/kernel/fp_sharing/src/float_regs_arc_gcc.h

```diff
@@ -0,0 +1,114 @@
+/**
+ * @file
+ * @brief ARC GCC specific floating point register macros
+ */
+
+/*
+ * Copyright (c) 2019, Synopys.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _FLOAT_REGS_ARM_GCC_H
```

Comment:
copy-paste mistake, please change into _ARC_ to avoid confusion.

Reply:
Fixed